File: PRAK302_2310817210022_MUHAMMAD_ADH-DHIYA_US_SALIM.c
#include <stdio.h>
int main() {
    int nilai;
    for(int i=0;i<5;i++){
    scanf("%d", &nilai);
    if(nilai >= 80){printf("A\n");}
    else if(nilai >= 70){printf("B\n");}
    else if(nilai >= 60){printf("C\n");}
    else if(nilai >= 50){printf("D\n");}
    else if(nilai <= 50){printf("E\n");}
    }
    return 0;
}

File: PRAK302_2310817210022_MUHAMMAD_ADH-DHIYA_US_SALIM.py
for i in range (0, 5):
    nilai = int(input())
    if nilai >= 80:
        print("A")
    elif nilai >= 70:
        print("B")
    elif nilai >= 60:
        print("C")
    elif nilai >= 50:
        print("D")
    elif nilai <= 50:
        print("E")
    else:
        print("Nilai yang dimasukkan tidak valid")